benchmark/atoi.cpp

```diff
@@ -116,7 +115,7 @@ std::size_t leadingSpacesCountAligned(S bytes) noexcept {
     auto otherWhiteSpace = belowEqualCarriageReturn & ~belowTab;
     auto whiteSpace = space | otherWhiteSpace;
     auto notWhiteSpace = ~whiteSpace;
-    auto rv = notWhiteSpace ? notWhiteSpace.lsbIndex() : S::Lanes;
```

Comment:
Why is this necessary?
`notWhiteSpace` ought to be a `BooleanSWAR`, this would not compile if it wasn't in a any platform.